packages/components/src/mobile/bottom-sheet-text-control/README.md

```diff
@@ -31,13 +31,12 @@ const MyTextControl = () => {
 				placeholder={ __( 'Generic placeholder text' ) }
 				label={ __( 'Generic label' ) }
 				icon={ textColor }
-				footerNote={
+				help={
 					<>
 						{ __( 'A footer note to add to the component! ' ) }
-						<FooterMessageLink
-							href={ 'https://wordpress.org/' }
-							value={ __( 'Visit WordPress.org' ) }
-						/>
+						<ExternalLink href={ 'https://wordpress.org/' } >
+							{ __( 'Visit WordPress.org' ) }
+						</ExternalLink>
 					</>
 				}
```

Comment:
While the original "Object Object" a11y label issue appears to be resolved in the app, I believe example offered here would still result in a problematic a11y label. We likely need to replace the example showcasing an inline link with the usage of `HelpText` [showcased in the changes this PR makes to the Image edit component](https://github.com/WordPress/gutenberg/blob/52d62bcc6a4f21363f48e86e5d627ff1ac9e8532/packages/block-library/src/image/edit.native.js#L576-L587) or change the entire architecture of this PRs approach to composing help texts with links.